src/components/reservations/ReservationCancelModal.tsx

```diff
@@ -37,14 +43,28 @@ const ReservationCancelModal = (): JSX.Element | null => {
 
   const closeDialog = () => dispatch(hideReservationCancelModal());
 
-  const handleFormCallback = (formData: ReservationCancelFormData) => {
-    setIsLoading(true);
-    console.log(formData); // TODO: Add operations here
-    setIsLoading(false);
-    closeDialog();
+  const handleFormCallback = async (formData: ReservationCancelFormData) => {
+    if (!postReservationCancelLoading) {
+      setIsLoading(true);
+
+      try {
+        // Send reservation cancel form data to API
+        await cancelApartmentReservation({ formData, reservationId: reservation.id, projectId: projectId })
+          .unwrap()
+          .then(() => {
+            toast.show({ type: 'success', content: t(`${T_PATH}.cancelledSuccessfully`) });
```

Comment:
I think this is missing from translations?

Reply:
Yeah, I noticed that after doing the reservation creation feature and it is included in that PR #57 already.

Reply:
Added the translations